pkg/label/annotation.go

```diff
@@ -0,0 +1,9 @@
+// Package label contains common Kubernetes metadata. These are defined in
+// https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md.
+package label
```

Comment:
I don't think changing the package name makes sense but I have added a separate file.